docs/resource.rst

```diff
@@ -1,21 +1,19 @@
-#################
-API Documentation
-#################
+##################
+Resource endpoints
+##################
 
-.. _http-apis:
+.. _resource-endpoints:
 
 GET /articles
 =============
 
-**Requires an FxA OAuth authentication**
+**Requires authentication**
 
-Returns all articles of the current user.
+Returns all records of the current user for this resource.
```

Comment:
I don't see why mentionning records would be a problem. If we keep articles, then this PR is not so relevant :)